src/main/java/org/folio/rest/impl/PurchaseOrderLineHelper.java

```diff
@@ -990,10 +990,14 @@ private boolean isISBN(String isbnTypeId, ProductId productId) {
     return Objects.equals(productId.getProductIdType(), isbnTypeId);
   }
 
-  private CompletableFuture<Void> checkLocationsAndPiecesConsistency(CompositePoLine poLine) {
-    String query = "poLineId==" + poLine.getId();
-    return piecesHelper.getPieces(Integer.MAX_VALUE, 0, query)
-      .thenAccept(pieces -> verifyLocationsAndPiecesConsistency(Collections.singletonList(poLine), pieces));
+  private CompletableFuture<Void> checkLocationsAndPiecesConsistency(CompositePoLine poLine, CompositePurchaseOrder order) {
+    if (order.getWorkflowStatus() == OPEN && Boolean.FALSE.equals(poLine.getCheckinItems())) {
```

Comment:
Lines with `receivingStatus=="Receipt not required"` and `isPackage==true` also have to be excluded from verification

Reply:
Done. Thanks!